mondrian/src/main/java/mondrian/parser/MdxParser.jj

```diff
@@ -992,6 +998,10 @@ QueryPart statement() :
 }
 {
     (
+        LOOKAHEAD(dmvSelectStatement())
+        //TODO: look ahead for <$SYSTEM>
```

Comment:
Can the TODO be addressed if it has something to do with line 246?

Reply:
I'll remove it for now, if we have performance issues we can revisit and figure out a more efficient look ahead strategy